.github/workflows/ci.yml

```
@@ -46,10 +46,33 @@ jobs:
         run: |
           pip3 install -r requirements.txt
           pip3 install --pre torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
+          pip3 install cmake ninja pybind11 pybind11-global
+          echo "/home/runner/.local/bin" >> $GITHUB_PATH
       - name: Install test requirements
         run: pip3 install expecttest fsspec iopath==0.1.9 numpy pytest rarfile
+      - name: Install AWS-SDK-CPP on Windows for S3 IO datapipes
+        if: ${{ matrix.os }} == 'windows-latest'
```

```suggestion
if: matrix.os == 'windows-latest'
```
I never get used to the syntax of github actions.

---

Edited and pushed